src/index.js

```
@@ -1,46 +1,38 @@
 import { useEffect, useState, useMemo } from "react";
-import algoliasearch from "algoliasearch";
 
 import {
   transformHit,
-  formatInputValue as defaultFormatInputValue,
+  formatInputValue as defaultFormatInputValue
 } from "./utils";
 
 const noop = () => {};
 
 const AlgoliaPlaces = ({
-  apiKey,
-  appId,
   children,
   defaultValue,
   formatInputValue = defaultFormatInputValue,
-  hitTransformer = transformHit,
   onSelect = noop,
   render,
+  searchClient,
```

maybe throw an error to let the user know that `searchClient` is required?